ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -146,47 +165,45 @@ def list_entities(self) -> Iterable[Table]:
 
         Same with `schema_filter_pattern`.
         """
-
-        # First, get all the databases for the service:
-        all_dbs = self.metadata.list_all_entities(
-            entity=Database,
-            params={"service": self.config.source.serviceName},
+        all_tables = self.metadata.list_all_entities(
+            entity=Table,
+            fields=[
+                "tableProfile",
+                "tests",
+            ],  # We will need it for window metrics to check past data
+            params={
+                "database": f"{self.config.source.serviceName}.{database.name.__root__}"
```

Comment:
here we could still keep the `service` parameter. Also, when building FQNs let's use `fqn.build`